src/movies.js

```diff
@@ -1,25 +1,100 @@
+
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    const allDirectors = moviesArray.map((movie) => {
+        return movie.director;
+    })
+    
+    return allDirectors;
```

Comment:
It's all good here, but keep in mind that since it is a direct return, you didn't need to create the allDirectors variable and could just return the map